types/ember-cli-showdown/markdown-to-html.d.ts

```diff
@@ -0,0 +1,222 @@
+    tasklists: boolean;
+
+    /**
+     * Prevents weird effects in live previews due to incomplete input
+     * @default false
+     */
+    smoothLivePreview: boolean;
+
+    /**
+     * Tries to smartly fix indentation problems related to es6 template strings in the midst of indented code.
+     * @default false
+     */
+    smartIndentationFix: boolean;
+
+    /**
+     * Disables the requirement of indenting sublists by 4 spaces for them to be nested, effectively reverting to the old behavior where 2 or 3 spaces were enough.
+     * @default false
+     */
+    disableForced4SpacesIndentedSublists: boolean;
+
+    /**
+     * Parses line breaks as <br>, without needing 2 spaces at the end of the line.
+     * @default false
+     */
+    simpleLineBreaks: boolean;
+
+    /**
+     * Makes adding a space between # and the header text mandatory
+     * @default false
+     */
+    requireSpaceBeforeHeadingText: boolean;
+
+    /**
+     * Enables github @mentions, which link to the username mentioned
+     * @default false
+     */
+    ghMentions: boolean;
+
+    /**
+     * Changes the link generated by @mentions. Showdown will replace {u} with the username. Only applies if ghMentions option is enabled. Example: @tivie with ghMentionsOption set to //mysite.com/{u}/profile will result in <a href="//mysite.com/tivie/profile">@tivie</a>
+     * @default https://github.com/{u}
+     */
+    ghMentionsLink: string;
+
+    /**
+     * Enable e-mail addresses encoding through the use of Character Entities, transforming ASCII e-mail addresses into its equivalent decimal entities.
+     * @default true
+     */
+    encodeEmails: boolean;
+
+
+    /**
+     * Open all links in new windows (by adding the attribute target="_blank" to <a> tags)
+     * @default false
+     */
+    openLinksInNewWindow: boolean;
+
+    /**
+     * Support for HTML Tag escaping. ex: \<div>foo\</div>
+     * @default false
+     */
+    backslashEscapesHTMLTags: boolean;
+
+    /**
+     * Enable emoji support. Ex: this is a :smile: emoji For more info on available emojis, see https://github.com/showdownjs/showdown/wiki/Emojis
+     * @default false
+     */
+    emoji: boolean;
+
+    /**
+     * EXPERIMENTAL FEATURE Enable support for underline. Syntax is double or triple underscores ex: __underlined word__. With this option enabled, underscores are no longer parses into <em> and <strong>.
+     * @default false
+     */
+    underline: boolean;
+
+    /**
+     * Outputs a complete html document, including <html>, <head> and <body> tags' instead of an HTML fragment.
+     * @default false
+     */
+    completeHTMLDocument: boolean;
+
+    /**
+     * Enable support for document metadata (defined at the top of the document between ««« and »»» or between --- and ---).
+     * @default false
+     */
+    metadata: boolean;
+
+    /**
+     * Split adjacent blockquote blocks.
+     * @default false
+     */
+    splitAdjacentBlockquotes: boolean;
+  };
+  Yields: {};
```

Comment:
If there is no `Yields`, do you need to specify the empty object?

Reply:
Fixed